scrnaseq.Rmd

```diff
@@ -1729,15 +1841,18 @@ We export the assay data, clustering, visualisation, marker genes and enriched p
 # top expressed genes
 sc = cerebroApp::getMostExpressedGenes(sc, column_cluster="seurat_clusters", column_sample="orig.ident", assay=Seurat::DefaultAssay(sc))
```

Comment:
Error in cerebroApp::getMostExpressedGenes(sc, column_cluster = "seurat_clusters",  : 
  unused arguments (column_cluster = "seurat_clusters", column_sample = "orig.ident")

but probably due to updated version of cerebroApp

Reply:
Well, what should I do with this one? Ignore? Would you like to re-run with an updated Cerebro package version for a test?

Reply:
It run through for me. So I will deal with this, once I observe the error too.